.circleci/config.yml

```diff
@@ -68,3 +80,9 @@ workflows:
           filters:
             branches:
               only: develop
+      - deploy_prod:
```

Comment:
On avait évoqué une validation manuelle possible, cf. https://circleci.com/docs/workflows#holding-a-workflow-for-a-manual-approval

Que pense tu d'ajouter ça au workflow de déploiement en prod et de la même façon permettre au dev de déployer la branche master en staging après un approval ?

Reply:
Très bonne idée, je ne savais pas que c'était possible _directement_ dans Circle CI mais du carrément partant! => j'ajoute ça 💡 

(dans mon esprit la validation manuelle ~= un petit message sur mattermost pour éliminer le doute)

Reply:
👍